frontend/src/scenes/cohorts/Cohort.tsx

```diff
@@ -1,199 +1,300 @@
+                                <LemonButton
+                                    data-attr="delete-cohort"
+                                    status="danger"
+                                    type="secondary"
+                                    onClick={() => {
+                                        deleteCohort()
+                                    }}
+                                    style={{ marginRight: 8 }}
+                                >
+                                    Delete
+                                </LemonButton>
+                            )}
+                            <LemonButton type="primary" data-attr="save-cohort" htmlType="submit">
+                                Save
                             </LemonButton>
-                        )}
-                        <LemonButton
-                            type="primary"
-                            data-attr="save-cohort"
-                            htmlType="submit"
-                            onClick={() => {
-                                saveCohort()
-                            }}
-                        >
-                            Save {cohort.id === 'new' ? 'cohort' : 'changes'}
-                        </LemonButton>
-                    </div>
-                }
-            />
-            <Divider />
-            <Row gutter={16}>
-                <Col md={14}>
-                    <CohortNameInput input={cohort.name} onChange={(name: string) => setCohort({ ...cohort, name })} />
-                </Col>
-                <Col md={10}>
-                    {cohort.id === 'new' ? (
-                        cohortTypeDropdown()
-                    ) : (
-                        <Tooltip title="Create a new cohort to use a different type of cohort.">
-                            <div>{cohortTypeDropdown()}</div>
-                        </Tooltip>
-                    )}
-                </Col>
-            </Row>
-            {hasAvailableFeature(AvailableFeature.DASHBOARD_COLLABORATION) && (
-                <Row gutter={16} className="mt">
-                    <Col span={24}>
-                        <CohortDescriptionInput description={cohort.description} onChange={onDescriptionChange} />
+                        </div>
+                    }
+                />
+                <Divider />
+                <Row gutter={[16, 24]} style={{ maxWidth: 640 }}>
+                    <Col xs={24} sm={12}>
+                        <Field name="name" label="Name">
+                            {({ value, onValueChange }) => <LemonInput value={value} onChange={onValueChange} />}
+                        </Field>
+                    </Col>
+                    <Col xs={24} sm={12}>
+                        <Field name="is_static" label="Type">
+                            {({ value, onValueChange }) => (
+                                <LemonSelect
+                                    disabled={!isNewCohort}
```

Comment:
I think we should keep the tooltip explaining why you can't change this after the cohort has been created.